common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -1505,4 +1536,4 @@ div.problem .annotation-input {
   .partially-correct {
     background: url('#{$static-path}/images/partially-correct-icon.png') center center no-repeat;
   }
-}
+}
```

Comment:
Looks like the newline at the end of the file was removed - all files should end in a newline.